airbase-policy/pom.xml

```diff
@@ -14,5 +14,6 @@
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <project.build.outputTimestamp>2024-06-05T18:08:46Z</project.build.outputTimestamp>
```

Comment:
can `project.build.outputTimestamp` be injected only for releases?

Reply:
Why are you asking?